src/Appwrite/Platform/Workers/Certificates.php

```diff
@@ -48,7 +48,11 @@ public function __construct()
             ->inject('queueForEvents')
             ->inject('queueForFunctions')
             ->inject('log')
-            ->callback(fn (Message $message, Database $dbForConsole, Mail $queueForMails, Event $queueForEvents, Func $queueForFunctions, Log $log) => $this->action($message, $dbForConsole, $queueForMails, $queueForEvents, $queueForFunctions, $log));
+            ->inject('adapterForCertificates')
+            ->callback(
+                fn (Message $message, Database $dbForConsole, Mail $queueForMails, Event $queueForEvents, Func $queueForFunctions, Log $log, AdapterProvider $adapterForCertificates) =>
```

Comment:
Why do we need an AdapterProvider wrapper? The type of `$adapterForCertificates` can just be `Appwrite\Certificates\Adapter` right? Just like we do for [`Database`](https://github.com/utopia-php/database/blob/main/src/Database/Database.php)